# Number of blocks on table
# int64 number_blocks

# Gripper open (0) or closed (1)
int64 gripper_state 

# Initial stack (0) or final stack (1)
int64 stack

# Fingers around block (1) or not (0)
# int64 to_block

# Gripper over a block (1) or not (0)
# int64 over_block

